src/v/kafka/server/errors.h

```diff
@@ -47,6 +47,8 @@ constexpr error_code map_topic_error_code(cluster::errc code) {
         return error_code::invalid_request;
     case cluster::errc::throttling_quota_exceeded:
         return error_code::throttling_quota_exceeded;
+    case cluster::errc::no_update_in_progress:
+        return error_code::no_reassignment_in_progress;
```

Comment:
Question, what are the program-wide implication(s) of making `no_update_in_progress` map to `no_reassignment_in_progress`? Could other requests now be returning a different error code then it would have before?

Reply:
you are correct. This change could introduce a regression. I put this in temporarily to see if any tests in CI fail.